### ISSUE TITLE
Sensor names not appearing correctly

### ISSUE THREAD

**Issue by brianleesmith** [isalkind/homebridge-away-mode#16]
When creating sensors, the individual names of the sensors are not showing up in HK.  They are showing up as the name of the master switch.  Running 1.6.1 version of plugin.  I’ve cleared the config and redone it with a single sensor, but it still shows up as the name of the master switch.

![03D125F8-6E0F-41D7-81EE-C96EE2500234](https://user-images.githubusercontent.com/17169723/206859009-426ec6ae-722c-4c41-8e6a-ddc4a3eb6200.png)
![654B760C-569D-4BBE-B96D-9BB5DC443EB3](https://user-images.githubusercontent.com/17169723/206859014-ca320805-c8fe-4b9e-864e-b80127d839ea.png)

**Comment by isalkind** [comment]
@brianleesmith Thanks for the tipoff. I can confirm that my setups are also showing the behavior that you are seeing. I'm guessing something deep in the bowels of HomeKit changed with the recent releases. I'll dig into it further when I have a chance to troubleshoot more.

I'm guessing everything is still completely functional. It's just hard to differentiate the sensors when they all end up with the same name. I would try adding the sensors one at at time so you can set up the appropriate automations as you go.

**Comment by FezVrasta** [comment]
Another issue is that you can't assign them to different rooms, they all end up in the same one.

**Comment by isalkind** [comment]
@brianleesmith As I suspected, this is due to a change in HomeKit. See https://github.com/homebridge/homebridge/issues/3210#issuecomment-1217735865.

To summarize:
* All services that are part of the accessory (in this case 'Away Mode') are given the same name.
* If you erase the name it assigns it will show the name you are wanting as a placeholder so you know what to set. So, attempting to change the name in the Home app will reveal what should have been set in the first place.

Closing as not an issue with the homebridge-away-mode plugin.

@FezVrasta Best practice would be to open a separate issue for a different problem. This will allow others who encounter the same issue to easily find the discussion on that topic.
